### ISSUE TITLE
Add more properties to nodes (and nodes.tsv) like aliases -- especially for ORFs

### ISSUE THREAD

**Issue by realmarcin** [Knowledge-Graph-Hub/kg-covid-19#225]

**Comment by cmungall** [comment]
See the current list broken down by category:

SELECT (COUNT(*) AS ?v2) ?v0 ?v1 WHERE {?v3 <https://w3id.org/biolink/vocab/category> ?v0 . ?v3 ?v1 ?v4} GROUP BY ?v0 ?v1

**Comment by cmungall** [comment]
this is better, limit to node properties:

```
SELECT (COUNT(*) AS ?v2) ?v0 ?v1 WHERE {?v3 <https://w3id.org/biolink/vocab/category> ?v0 . ?v3 ?v1 ?v4 . FILTER isLITERAL(?v4)} GROUP BY ?v0 ?v1

1       gene_ontology   blmod:category
1       gene_ontology   blmod:description
1       gene_ontology   rdfs:label
1       blmod:OntologyClass     tmp:tchem
1       blmod:OntologyClass     tmp:tclin
1       blmod:MolecularEntity   tmp:provided_by
1       blmod:MolecularEntity   tmp:ncbi_taxid
1       blmod:MolecularEntity   rdfs:label
2       gene_ontology   tmp:provided_by
4       blmod:ChemicalSubstance tmp:ncbi_taxid
4       blmod:Drug      tmp:ncbi_taxid
6       blmod:OrganismalEntity  rdfs:label
7       blmod:OrganismalEntity  tmp:provided_by
7       blmod:RNA       tmp:provided_by
7       blmod:RNA       tmp:ncbi_taxid
7       blmod:RNA       rdfs:label
8       external        blmod:synonym
33      blmod:OntologyClass     tmp:ncbi_taxid
46      sequence        blmod:category
46      sequence        blmod:description
46      sequence        tmp:provided_by
46      sequence        rdfs:label
50      blmod:Protein   blmod:synonym
64      external        blmod:description
76      sequence        blmod:synonym
87      external        blmod:category
87      external        tmp:provided_by
87      external        rdfs:label
108     blmod:Gene      tmp:TTD_ID
129     blmod:Gene      tmp:ncbi_taxid
135     blmod:Disease   blmod:description
177     blmod:Protein   tmp:tchem
177     blmod:Protein   tmp:tclin
195     blmod:Disease   blmod:synonym
219     blmod:Protein   tmp:TTD_ID
225     blmod:NamedThing        rdfs:label
227     blmod:NamedThing        tmp:provided_by
289     blmod:Disease   tmp:provided_by
289     blmod:Disease   rdfs:label
1151    blmod:Protein   tmp:ncbi_taxid
1575    blmod:Gene      tmp:tchem
1575    blmod:Gene      tmp:tclin
1756    blmod:Protein   rdfs:label
2215    blmod:Drug      tmp:tchem
2215    blmod:Drug      tmp:tclin
3543    blmod:OntologyClass     blmod:category
4330    blmod:AnatomicalEntity  blmod:description
4454    blmod:CellularComponent blmod:description
4454    blmod:CellularComponent rdfs:label
4656    blmod:AnatomicalEntity  tmp:provided_by
4656    blmod:AnatomicalEntity  rdfs:label
4925    blmod:CellularComponent blmod:synonym
5652    blmod:CellularComponent tmp:provided_by
8043    human_phenotype blmod:description
10384   human_phenotype blmod:category
10384   human_phenotype rdfs:label
12202   blmod:MolecularActivity blmod:description
12202   blmod:MolecularActivity rdfs:label
13926   human_phenotype tmp:provided_by
14928   blmod:MolecularActivity tmp:provided_by
16159   blmod:AnatomicalEntity  blmod:synonym
16612   human_phenotype blmod:synonym
17812   blmod:Gene      blmod:description
19577   blmod:Protein   tmp:provided_by
21169   blmod:Gene      tmp:provided_by
22952   blmod:ChemicalSubstance rdfs:label
23075   blmod:ChemicalSubstance tmp:provided_by
24883   blmod:Gene      rdfs:label
28870   blmod:Drug      tmp:TTD_ID
29127   blmod:MolecularActivity blmod:synonym
30702   blmod:BiologicalProcess blmod:description
30702   blmod:BiologicalProcess rdfs:label
32194   blmod:Drug      rdfs:label
32326   blmod:Drug      tmp:provided_by
32624   blmod:OntologyClass     blmod:description
35099   blmod:BiologicalProcess tmp:provided_by
42903   blmod:OntologyClass     rdfs:label
48232   blmod:Publication       rdfs:label
52097   blmod:Publication       tmp:provided_by
94066   blmod:BiologicalProcess blmod:synonym
142558  blmod:OntologyClass     blmod:synonym
163453  blmod:OntologyClass     tmp:provided_by

```

**Comment by cmungall** [comment]
we could potentially start using bl toolkit to rather than hardcode strings but not urgent:

https://github.com/Knowledge-Graph-Hub/kg-covid-19/blob/9cab8b413db18745ac438c4eddabafdf73bad80e/kg_covid_19/transform_utils/sars_cov_2_gene_annot/sars_cov_2_gene_annot.py#L24

**Comment by justaddcoffee** [comment]
Closed by #232
